### PR TITLE
Change links

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

File: tyk-docs/content/security/security-policies/policies-guide.md [hunk 1]
```diff
@@ -139,21 +139,17 @@ Although we have set the main factors of the key, they will be overridden by the
 
 ## <a name="with-api"></a>Create a policy with the Dashboard API
 
-On how to create a policy with the Dashboard API check [this section][1]
+[See here](https://tyk.io/docs/get-started/with-tyk-on-premise/tutorials/tyk-on-premises-pro/create-security-policy/#a-namewithapiatutorial-create-a-security-policy-with-the-api)
```
Comment (asoorm) [inline tyk-docs/content/security/security-policies/policies-guide.md]: Change anchor text from `See here` to `tutorial create security policy via api`

File: tyk-docs/content/security/security-policies/policies-guide.md [hunk 2]
```diff
@@ -139,21 +139,17 @@ Although we have set the main factors of the key, they will be overridden by the
...
 
 ## <a name="create-a-file-based-policy"></a>Create a policy with the Gateway API
 
-On how to create a policy with the Gateway API check [this section][2]
+[See here](https://tyk.io/docs/with-tyk-community-edition/tutorials/create-security-policy/#a-name-create-a-file-based-policy-a-tutorial-create-a-policy-with-the-gateway-api).
```
Comment (asoorm) [inline tyk-docs/content/security/security-policies/policies-guide.md]: same as above

File: tyk-docs/content/security/security-policies/policies-guide.md [hunk 3]
```diff
@@ -139,21 +139,17 @@ Although we have set the main factors of the key, they will be overridden by the
...
 
 ## <a name="with-dashboard"></a>Create a policy with the Dashboard
 
-On how to create a policy with the Gateway API check [this section][3]
+[See here](https://tyk.io/docs/get-started/with-tyk-on-premise/tutorials/tyk-on-premises-pro/create-security-policy/#a-namewithdashboardatutorial-create-a-security-policy-with-the-dashboard)
```
Comment (asoorm) [inline tyk-docs/content/security/security-policies/policies-guide.md]: same as above

**Review by asoorm — CHANGES_REQUESTED** [review]
Anchor links should provide context & description - rather than `see here` or similar.

**Comment by marksou** [comment]
@asoorm I've changes the links section completely.

**Review by asoorm — APPROVED** [review]

**Pull request merged**